community-operators/okd-windows-machine-config-operator/0.0.1/okd-windows-machine-config-operator.0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,328 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+    categories: "OpenShift Optional"
+    certified: "false"
+    containerImage: quay.io/openshift-windows/okd-windows-machine-config-operator:4.6-1d646e8
+    createdAt: 2020-09-01T11:59:59Z
+    description: An operator that enables Windows container workloads on OKD
+    operatorframework.io/suggested-namespace: windows-machine-config-operator
+    repository: https://github.com/openshift/windows-machine-config-operator
+    support: ""
+  name: okd-windows-machine-config-operator.v0.0.1
+  namespace: windows-machine-config-operator
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned: []
+  description: |-
+    # OKD Windows Machine Config Operator
+
+    ## WARNING
+    Community distribution of the Windows Machine Config Operator. This is an alpha build, and is not meant for
+    production workloads. Issues with this distribution of WMCO can be made against the [WMCO repository](https://github.com/openshift/windows-machine-config-operator).
+
+    Please ensure that when installing this operator the startingCSV you subscribe to matches is supported on the
+    version of OKD you are using. This CSV is meant for OKD 4.6.
+
+    Upgrading the OKD cluster from version 4.6 to 4.7 while running this operator is not supported. When upgrading
+    OKD, remove all Windows machinesets and uninstall this operator. Once the OKD upgrade process has completed,
```

Comment:
I think this should be reworded. Upgrading OKD cluster with Windows Machines running will not result in Windows Machines being upgraded with latest OKD components. As such, the cluster upgrade should work fine right?